src/constants/numbers.ts

```diff
@@ -1,6 +1,20 @@
+import * as constants from '@ethersproject/constants'
+import { BigNumber } from '@ethersproject/bignumber'
+
 export const WAD_DECIMALS = 18
 
 export const SECONDS_IN_DAY = 24 * 60 * 60
 export const SECONDS_IN_HOUR = 60 * 60
 
 export const PROJECT_PAY_CHARACTER_LIMIT = 16
+
+export const TEN_THOUSAND = 10000
+export const ONE_MILLION = 1000000
+export const ONE_BILLION = 1000000000
+
+export const MaxUint232 = constants.MaxUint256.add(1)
+  .div(2 ** 24)
+  .sub(1)
+export const MaxUint88 = 2 ** 88 - 1
+export const MaxUint54 = BigNumber.from(10).pow(54).sub(1)
```

Comment:
What is different about this one?

Reply:
it just has `BigNumber` type. `BigNumber.from(2**55 - 1)` is a runtime exception 😢 .

I know its bad for consistency, but... eh

Reply:
Shouldn't it be `BigNumber.from(2).pow(54).sub(1)` ?

Reply:
derp. yes. changed. thank you 🙏